HW15/src/FileLoggerConfiguration.java

```diff
@@ -0,0 +1,83 @@
+import java.io.File;
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.Date;
+
+public class FileLoggerConfiguration {
+
+    private static double maxFileSize = 0.3;
+    private static String filePath = "src/";
```

Comment:
Under the src folder must be only your java classes. Configuration files must be in the `resource` folder or the project's root.